<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
background-color: rgb(42, 39, 41);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма кредитования:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Срок кредитования:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Общая сумма выплаты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Переплата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>мес.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Тип кредитования:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетный (F1)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный (F2)</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_sum_credit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_time_credit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_procent">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_sum_pay">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_over_pay">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_month_pay">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_res">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="err">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 35pt;
background-color: rgb(42, 39, 41);
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
